web/src/modals.ts

```diff
@@ -32,6 +32,10 @@ function call_hooks(func_list: Hook[]): void {
     }
 }
 
+export function any_active_or_animating(): boolean {
+    return $(".micromodal").hasClass("modal--open") || $(".micromodal").hasClass("modal--opening");
+}
```

Comment:
We can make this check faster by combining both the queries.

Reply:
https://stackoverflow.com/a/7490281
Says that combining it using `is` will make it slower. Is there a better way?

Reply:
You could at least avoid doing `$(".micromodal")` twice.